Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -142,6 +154,69 @@ private fun <T> ThreePaneScaffoldNavigator<T>.isMainPaneHidden(): Boolean {
     return scaffoldValue[SupportingPaneScaffoldRole.Main] == PaneAdaptedValue.Hidden
 }
 
+@OptIn(ExperimentalMaterial3AdaptiveApi::class)
+fun calculateScaffoldDirective(
```

Comment:
Would be good to add a comment on how this differs from the default `calculateScaffoldDirective`

Reply:
Done